src/Filter/Sections/TeaserSection.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
Is cloning the whole importer here necessary? Only checking if the teaser is already present and link to that one might be sufficient. If duplication is necessary, at least make sure the ignore on duplicatze check is working as with the normal import.

Reply:
We need it as the teaser import needs to stay configureable, and has the same configurations as the normal importer.

Reply:
As discussed. We go the lazy route for internal Teasers with sugs.